engines/zypper_auth/lib/zypper_auth/engine.rb

```diff
@@ -100,20 +116,22 @@ def make_repo_url(base_url, repo_local_path, service_name)
           "plugin:/susecloud?credentials=#{service_name}&path=" + url.path
         end
 
+        # additional validation for zypper service XML controller
         before_action :verify_instance
         def verify_instance
-          unless ZypperAuth.verify_instance(request, logger, @system)
-            render(xml: { error: 'Instance verification failed' }, status: 403)
-          end
+          ZypperAuth.verify_instance(request, logger, @system)
+          true # don't raise auth errors (for now)
```

Comment:
Where do auth errors get raised now?

Reply:
@bear454 No errors until we switch. Raising auth errors was pre-mature. This is coming back at the time it is agree that we want strict authentication. With the current implementation we are in an "in between state" wich is not correct. We want either all access strictly enforced or none.